src/IndexingWrapper.cpp

```diff
@@ -137,6 +138,7 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
     if (id < 0 || id > dfCols) Rcpp::stop("Invalid argument for indexCol");
     Rcpp::List indexList = indexFields;
     Rcpp::List filterList = filterFields;
+    Rcpp::List slotList = valueSlots;
```

Comment:
>
> Is there a reason to do this?


I do it for convenience during development. Because otherwise if I change
the name of a parameter, I've to go back and change the name in all the
places. (RStudio doesn't provide automatic renaming support)

This may be true for some of the other parameters as well
>
Will change those as a separate PR.

On Fri, Jul 1, 2016 at 5:19 PM, James Aylett <notifications@github.com>
wrote:

> In src/IndexingWrapper.cpp
> <https://github.com/amandaJayanetti/RXapian/pull/10#discussion_r69287828>:
>
> > @@ -137,6 +138,7 @@ indexWrapper(Rcpp::CharacterVector & dbpath, Rcpp::DataFrame & dataFrame, Rcpp::
> >      if (id < 0 || id > dfCols) Rcpp::stop("Invalid argument for indexCol");
> >      Rcpp::List indexList = indexFields;
> >      Rcpp::List filterList = filterFields;
> > +    Rcpp::List slotList = valueSlots;
>
> Is there a reason to do this? I'd have thought you could operate directly
> on valueSlots. (This may be true for some of the other parameters as
> well.)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/amandaJayanetti/RXapian/pull/10/files/4e02e68fe0801ec7bf4aafdf8db90225372e1a29#r69287828>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AH_jNrNLRv4epeD56SEvCb7HQ2eCvwipks5qRP7GgaJpZM4JCweD>
> .
>